ConsoleAppn7/ConsoleAppn7/Program.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите X");
+        var x = int.Parse(Console.ReadLine());
+        Console.WriteLine($"полученное значение функции:" + Coordinate(x));
+        Console.ReadLine();
+    }
+    static double Coordinate(int x)
+    {
+        if (x > 0)
+            return Math.Pow(Math.E, -1 * x);
+        else if (x <= -1)
+            return Math.Pow(Math.E, x + 1);
```

Comment:
В классе Math для вычисления экспоненты имеется специальный метод:
```suggestion
            return Math.Exp(-x);
        else if (x <= -1)
            return Math..Exp(x + 1);
```